unlock-app/src/components/interface/VerificationStatus.tsx

```diff
@@ -173,12 +198,17 @@ export const VerificationStatus = ({ config, onVerified, onClose }: Props) => {
       })
     : 'Invalid QR code'
 
+  const isVerifier =
```

Comment:
I really don't understand that? 
Whhy would `isVerifier` be true if the the lock address is the same `address` ?